src/cascadia/TerminalSettingsModel/FolderEntry.cpp

```diff
@@ -48,7 +48,10 @@ winrt::com_ptr<NewTabMenuEntry> FolderEntry::FromJson(const Json::Value& json)
 
     JsonUtils::GetValueForKey(json, NameKey, entry->_Name);
     JsonUtils::GetValueForKey(json, IconKey, entry->_Icon);
-    JsonUtils::GetValueForKey(json, EntriesKey, entry->_Entries);
+    if (const auto& entries{ JsonUtils::GetValueForKey<decltype(entry->_Entries)>(json, EntriesKey) })

```

Comment:
Can you explain quick how this is different from `GetValueForKey(..., into)`? It seems largely the same, but more verbose and error-prone.

Reply:
@DHowett IIRC, the issue was that `GetValueForKey(..., into)` would happily place `nullptr` into `into` if it didn't find `entries`. So the later attempt to query it would a/v. 

I think I started here, and then added the null check, and I probably should have just started there. I'll revert these lines.